db/controllor/controller.js

```diff
@@ -18,6 +18,17 @@ const getArticles = (request, response, next) => {
     .catch((err) => {
         next(err);
     })
+};
+
+const postComment = (request, response, next) => {
+    const { article_id } = request.params;
```

Comment:
Since you're de-structuring the id here, it would be best practice to also de-structure the body at the same time in order to keep it consistent.